Выполнены задачи по заданию спринта №25

Neolight79 opened the pull request:
- Создан новый проект FirstComposeProject для демонстрации отображения карточки контакта средствами JetPack Compose фреймворка.
- Все задачи выполнены в модуле главной активити.
- Функция ContactDetails получает на входе DataClass с данными контакта и отображает данные карточки согласно предложенному макету.
- Оба варианта предложенных карточек контактов выводятся в режиме Preview.
- Один из вариантов карточки показывается в режиме запуска приложения.

Commit:
Activity structure prepared

Commit:
Activity structure prepared

Commit:
Activity structure prepared

Commit:
Выполнены задачи по заданию спринта №25
- Создан новый проект FirstComposeProject для демонстрации отображения карточки контакта
    средствами JetPack Compose фреймворка.
- Все задачи выполнены в модуле главной активити.
- Функция ContactDetails получает на входе DataClass с данными контакта и отображает данные карточки согласно предложенному макету.
- Оба варианта предложенных карточек контактов выводятся в режиме Preview.
- Один из вариантов карточки показывается в режиме запуска приложения.

anastasianadelshtehel commented on app/src/main/java/com/example/firstcomposeproject/MainActivity.kt:
```diff
@@ -34,121 +32,127 @@ class MainActivity : ComponentActivity() {
+                modifier = Modifier.height(64.dp),
                 contentAlignment = Alignment.Center
+            ){
+                when (contact.imageRes == null) {
```
Логика работы правильная, но можно сделать так
```
when {
    contact.imageRes == null -> { ... }
    else -> Image(...)
}
```

anastasianadelshtehel commented on app/src/main/java/com/example/firstcomposeproject/MainActivity.kt:
```diff
@@ -34,121 +32,127 @@ class MainActivity : ComponentActivity() {
+                        Text(
+                            text = contact.name.take(1) + contact.familyName.take(1),
+                            fontWeight = FontWeight.Bold,
+                            fontSize = 20.sp
```
Размеры шрифтов (16.sp, 20.sp) и отступы (16.dp, 8.dp) лучше вынести в константы

Neolight79 committed:
Выполнены задачи по заданию спринта №25 (UPD1)
- Создан новый проект FirstComposeProject для демонстрации отображения карточки контакта
    средствами JetPack Compose фреймворка.
- Все задачи выполнены в модуле главной активити.
- Функция ContactDetails получает на входе DataClass с данными контакта и отображает данные
    карточки согласно предложенному макету.
- Оба варианта предложенных карточек контактов выводятся в режиме Preview.
- Один из вариантов карточки показывается в режиме запуска приложения.
UPD1:
- Значения отступов и размеры шрифтов перенесены в ресурсы
- Скорректировано условие для блока when
